src/Pages/Events/EventList.js

```diff
@@ -19,23 +18,6 @@ function AnnouncementList() {
     if (!eventResponse.error) setEventList(eventResponse.responseData);
   }
 
-  const getFilteredEvents= () => {
-    if (getFiltered){
-      try {
-        let currDate = new Date();
-        currDate.setDate(currDate.getDate() -1);
-        let validList = [];
-        eventList.forEach(item => {
-          let date = new Date(item.eventDate);
-          if (date >= currDate) {
-            validList.push(item);
-          }
-        }, setValidList(validList), setGetFiltered(false));
-      } catch (error) {
-      }
-    }
-  };
-
```

Comment:
deleting this causes an error in the frontend-test for some reason

Reply:
i am changing the test for eventlist and refactoring it a bit